releases/release-6.5.7.md

```diff
@@ -0,0 +1,151 @@
+---
+title: TiDB 6.5.7 Release Notes
+summary: 了解 TiDB 6.5.7 版本的兼容性变更、改进提升，以及错误修复。
+---
+
+# TiDB 6.5.7 Release Notes
+
+发版日期：2023 年 x 月 x 日
+
+TiDB 版本：6.5.7
+
+试用链接：[快速体验](https://docs.pingcap.com/zh/tidb/v6.5/quick-start-with-tidb) | [生产部署](https://docs.pingcap.com/zh/tidb/v6.5/production-deployment-using-tiup) | [下载离线包](https://cn.pingcap.com/product-community/?version=v6.5.7#version-list)
+
+## 兼容性变更
+
+无
+
+## 改进提升
+
++ TiDB
+    <!-- tw@Oreoxmt -->
+    - (dup): release-7.4.0.md > 改进提升> TiDB - 优化 `ANALYZE` 分区表的内存使用和性能 [#47071](https://github.com/pingcap/tidb/issues/47071) [#47104](https://github.com/pingcap/tidb/issues/47104) [#46804](https://github.com/pingcap/tidb/issues/46804) @[hawkingrei](https://github.com/hawkingrei)
+    - Plan Cache 支持主键的 `IN (....)` 语法[#44830](https://github.com/pingcap/tidb/issues/44830) @[qw4990](https://github.com/qw4990)
+    - 提供 TiDB 等待统计信息加载完成才开始对外提供服务的开关 [#43385](https://github.com/pingcap/tidb/issues/43385) @[xuyifangreeneyes](https://github.com/xuyifangreeneyes)
```

Comment:
>提供 TiDB 等待统计信息加载完成才开始对外提供服务的开关 [#43385](https://github.com/pingcap/tidb/issues/43385) @[xuyifangreeneyes](https://github.com/xuyifangreeneyes)

@winoros Could you please help confirm if this enhancement introduces the force-init-stats configuration (https://github.com/pingcap/docs-cn/pull/13840) in v6.5.7?